qiskit-superstaq/qiskit_superstaq/daily_integration_test.py

```diff
@@ -247,7 +247,11 @@ def test_submit_to_provider_simulators(target: str, provider: qss.SuperstaqProvi
     assert job.result().get_counts() == {"11": 1}
 
 
-@pytest.mark.skip(reason="Can't be executed when Hilbert is set to not accept jobs")
+@pytest.mark.skipif(
+    "cq_hilbert_qpu"
+    not in qss.SuperstaqProvider()._client.get_targets()["superstaq_targets"]["compile-and-run"],
```

Comment:
See previous comment. Also, I think you should be able to call `get_targets()` directly from `qss.SuperstaqProvider()` since it inherits from `gss.service.Service()` which contains a `get_targets()` wrapper.